app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -126,6 +126,7 @@ def update_outgoing_email_status
     email_to_update = (
       OutgoingEmail.find_by(message_id: message_id) ||
         VerificationEmail.find_by(mailgun_id: message_id) ||
+        AssignmentEmail.find_by(message_id: message_id) ||
```

Comment:
I have a personal desire for us to have **less** tables that store a `message_id` / `mailgun_status`

otherwise we'll keep adding to this list for any new email we make, when they should probably be handled more uniformly

Is there a reason we can't try to cram this `assignment_email` into the existing `SendOutgoingEmailJob` / `OutgoingEmail` / `OutgoingEmailMailer` infrastructure?

I imagine the issue is that messages sent through those classes are targeted at clients rather than hub users, but we could maybe make it work? Or if there's a dealbreaker we could build a parallel `OutgoingHubUserEmail` pathway that is more generalizable for when we add new emails targeting hub users